code/game/objects/structures/crates_lockers/closets/cardboardbox.dm

```diff
@@ -62,3 +64,16 @@
 			for(var/mob/M in viewers(src))
 				M.show_message("<span class='notice'>\The [src] has been cut apart by [user] with \the [WC].</span>", 3, "You hear cutting.", 2)
 			qdel(src)
+			return
+		if(istype(W, /obj/item/weapon/storage/fancy/crayons)) /////addendum by flimflamm
+			var/decalselection = input("Please select a decal") in list("atmos", "bar", "barber", "blueshield",	"brigphysician", "captain",
+				"cargo", "ce",	"caplain",	"chef", "chemist", "civilian", "clown", "cmo", "coroner", "detective", "engineering", "genetics", "hop",
+				"hos", "hydroponics", "ia", "janitor",	"magistrate", "mechanic", "medical", "mime", "mining", "ntrep", "paramedic", "podpilot",
+				"prisoner",	"rd", "security", "syndicate", "therapist", "virology", "warden", "xenobiology")
+
+			icon_opened = addtext("cardboard_open_", decalselection)
+			icon_closed = addtext("cardboard_", decalselection)
```

Comment:
Oh, right, you need a sanity check for adjacency after the input. I always forget about those.